homeassistant/components/tesla_fleet/strings.json

```diff
@@ -412,6 +412,19 @@
       "vehicle_state_odometer": {
         "name": "Odometer"
       },
+      "island_status": {
+        "name": "Grid Status",
+        "state": {
+          "island_status_unknown": "Unknown",
+          "on_grid": "Connected",
+          "off_grid": "Disconnected",
+          "off_grid_unintentional": "Disconnected",
```

Comment:
Should this really be translated same as the one above?

Reply:
I don't think it should be, I'd suggest adding the intentional / unintentional aspect to the translation.

Reply:
are you talking about off_grid and off_grid_unintentional?

Reply:
If the key name contains `unintentional` I guess it would be logic to add that as a word to the translation.

Reply:
Resolved [d492c28](https://github.com/home-assistant/core/pull/126438/commits/d492c2853a95900020ae322b1b451ecdf5f8d535)